### PR TITLE
TIP-731: Adds missing config for single storage and webpack

### PR DESCRIPTION
**Pull request by tamarasaurus**
This PR makes the necessary changes to get single storage and webpack working

### CODE REVIEW

**Comment by tamarasaurus** [comment]
@anaelChardan I ran across another issue, does `security.exception_listener.class: Pim\Bundle\UserBundle\Security\FirewallExceptionListener` need to be removed? It doesn't look like that class exists anymore, it throws an error

File: UPGRADE-1.7.md [hunk 1]
```diff
@@ -1,422 +0,0 @@
-# UPGRADE FROM 1.6 to 1.7
```
Comment (damien-carcel) [inline UPGRADE-1.7.md]: Why did you remove this file?

Comment (tamarasaurus) [inline UPGRADE-1.7.md]: Not sure how the contents became empty, it's fixed now
